doc/tdshim_spec.md

```diff
@@ -441,6 +441,9 @@ second parameter.
    loaded.
  * All rest are same as above.
 
+Before TD Shim hands over control to payload, TD Shim will reset the Interrupt
```

Comment:
I dont think TD shim should reset IDT. It will still have an IDT point to reserved memory.

The payload should set its own IDT, same as other GDT, page table, etc.